test/images/volume/rbd/osd.sh

```diff
@@ -19,7 +19,7 @@
 #
 
 ceph osd create
-ceph-osd -i $1 --mkfs --mkkey
-ceph auth add osd.$1 osd 'allow *' mon 'allow rwx' -i /var/lib/ceph/osd/ceph-$1/keyring
-ceph osd crush add $1 1 root=default host=cephbox
-ceph-osd -i $1 -k /var/lib/ceph/osd/ceph-$1/keyring
+ceph-osd -i "${1}" --mkfs --mkkey
+ceph auth add "osd.${1}" osd 'allow *' mon 'allow rwx' -i "/var/lib/ceph/osd/ceph-${1}/keyring"
+ceph osd crush add "${1}" 1 root=default host=cephbox
+ceph-osd -i "${1}" -k "/var/lib/ceph/osd/ceph-${1}/keyring"
```

Comment:
why change:
```
In ./test/images/volume/rbd/osd.sh line 22:
ceph-osd -i $1 --mkfs --mkkey
            ^-- SC2086: Double quote to prevent globbing and word splitting.


In ./test/images/volume/rbd/osd.sh line 23:
ceph auth add osd.$1 osd 'allow *' mon 'allow rwx' -i /var/lib/ceph/osd/ceph-$1/keyring
                  ^-- SC2086: Double quote to prevent globbing and word splitting.
                                                                             ^-- SC2086: Double quote to prevent globbing and word splitting.


In ./test/images/volume/rbd/osd.sh line 24:
ceph osd crush add $1 1 root=default host=cephbox
                   ^-- SC2086: Double quote to prevent globbing and word splitting.


In ./test/images/volume/rbd/osd.sh line 25:
ceph-osd -i $1 -k /var/lib/ceph/osd/ceph-$1/keyring
            ^-- SC2086: Double quote to prevent globbing and word splitting.
                                         ^-- SC2086: Double quote to prevent globbing and word splitting.
```